resources/views/partials/menu-top.blade.php

```diff
@@ -30,6 +34,11 @@
                         </div>
                     </div>
                 </div>
+
+                @if(isset($banner) && $banner != false)
```

Comment:
If this is the trigger for the banner perhaps we add to the [`README.md`](https://github.com/waynestate/base-site/blob/master/README.md#developing-views) on how to trigger it from a controller?

Reply:
Its triggered via per page promotions in the `Banner` group defined in the `PromoRepository` -> https://github.com/waynestate/base-site/pull/23/files/bc7aacab6ce16520b316a1811729193728d11359#diff-6bd8c85afde6cd73327d6415a509969aR71

So we don't need to trigger it from a controller at all. I'm cool with expanding the documentation though and explaining how these global promotion groups work!

Reply:
Sounds good. I think documenting all the inherent features of the overall template would be a good next step.